test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorPageActivatorTest.cs

```diff
@@ -66,7 +66,7 @@ public void Activate_ActivatesAndContextualizesPropertiesOnViews()
             var viewContext = new ViewContext(
                 actionContext,
                 Mock.Of<IView>(),
-                new ViewDataDictionary(new EmptyModelMetadataProvider()),
+                new ViewDataDictionary(new EmptyModelMetadataProvider(), new ModelStateDictionary()),
```

Comment:
/cc @jbagga, Ajay took care of some of this for you to let him use shared sources.